app/constants/constants.ts

```diff
@@ -0,0 +1 @@
+export const apiEndPoint = "http://localhost:5213/playlist";
```

Comment:
you can put uri in .env file and constant in constants file

Reply:
updated code to get data from env